src/components/Dropdown.js

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import "./Dropdown.css";
+
+export default function Dropdown({ setParamObj }) {
+  const [isActive, setIsActive] = useState(false);
+  const [value, setValue] = useState("recent");
+
+  const handleSelectClick = () => {
+    setIsActive((prev) => !prev);
+  };
+  const handleSelectBlur = (e) => {
+    if (e.relatedTarget?.className === "orderSelect__option") return;
+    setIsActive(false);
+  };
+  const handleRecentClick = () => {
+    setValue("recent");
+    setIsActive(false);
+  };
+  const handleFavoriteClick = () => {
+    setValue("favorite");
+    setIsActive(false);
+  };
```

Comment:
P3:
하나의 함수로 사용하시면 더 좋을 것 같습니다.

```suggestion
  const handleClick = (e) => {
    setValue(e.target.value);
    setIsActive(false);
  };
```